README.md

```diff
@@ -43,12 +53,15 @@ To get a local copy up and running follow these simple example steps.
 
 ## Authors
 
-The rights over this page are reserved for [@Cindy_Shin] (https://www.behance.net/adagio07)
+👤 **Sergio Valadez**
 
-👤 **Author**
+- GitHub: [@githubhandle](https://github.com/sergiogval)
+- Twitter: [@twitterhandle](https://twitter.com/sgvldz)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
 
-- GitHub: [@sergiogval](https://github.com/sergiogval)
-- Instagram: [@sergioval](https://instagram.com/sergioval)
+👤 **Cindy Shin**
+
+- Original design idea
```

Comment:
- [ ] Kindly remove people who did not contribute to this project. This will make your readme more professional

Reply:
Done